src/main/java/com/hotels/bdp/circustrain/bigquery/extraction/BigQueryExtractionData.java

```diff
@@ -61,6 +80,11 @@ Table getTable() {
     return table;
   }
 
+  @Override
+  public String toString() {
+    return "Table: " + getDatasetName() + getTableName() + " Location: " + getDataUri();
```

Comment:
Is it ok to output the data set name and the table name without any kind of separator? What are typical values for them?

Reply:
There should be a "." will fix